providers/outlook365/src/lib/outlook365.provider.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  ChannelTypeEnum,
+  IEmailOptions,
+  IEmailProvider,
+  ISendMessageSuccessResponse,
+  ICheckIntegrationResponse,
+  CheckIntegrationResponseEnum,
+} from '@novu/stateless';
+import nodemailer, { SendMailOptions, Transporter } from 'nodemailer';
+
+export class Outlook365Provider implements IEmailProvider {
+  id = 'outlook365';
+  channelType = ChannelTypeEnum.EMAIL as ChannelTypeEnum.EMAIL;
+
+  private transports: Transporter;
+
+  constructor(
+    private config: {
+      user: string;
+      password: string;
+    }
+  ) {
+    this.transports = nodemailer.createTransport({
+      host: 'smtp.office365.com',
+      port: 587,
+      requireTLS: true,
+      connectionTImeout: 30000,
+      auth: {
+        user: this.config.user,
+        pass: this.config.password,
+      },
+      tls: {
+        ciphers: 'SSLv3',
+      },
+    });
+  }
+
+  async sendMessage(
+    options: IEmailOptions
+  ): Promise<ISendMessageSuccessResponse> {
+    const mailData = this.createMailData(options);
+    const info = await this.transports.sendMail(mailData);
+
+    return {
+      id: info?.messageId,
+      date: new Date().toISOString(),
```

Comment:
Could we check if in the response Outlook 365 sends any timestamp that we could reuse here instead of setting it this way?